packages/vitest/src/types/tasks.ts

```diff
@@ -21,6 +21,8 @@ export interface TaskResult {
   startTime?: number
   error?: ErrorWithDiff
   hooks?: Partial<Record<keyof SuiteHooks, TaskState>>
+  // only for ui: handling internal colored stack traces on report tab
+  uiHtmlError?: string
```

Comment:
What about:
```suggestion
  error?: ErrorWithDiff
  htmlError?: string
  hooks?: Partial<Record<keyof SuiteHooks, TaskState>>
```
`error` is already using something the UI could use, so why not just named it `htmlError`?